### PR TITLE
Update to flake8 v5

### PR DESCRIPTION
**Pull request by Uxio0**

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -1,7 +1,7 @@
 [flake8]
 max-line-length = 88
 select = C,E,F,W,B,B950
-extend-ignore = E203, E501, F841
+extend-ignore = E203,E501,F841,W503
```
Comment (moisses89) [inline setup.cfg:4]: commas without space? 
W503 should be ignored?

Comment (Uxio0) [reply setup.cfg:4]: It's INI format, not python 😉 , so better without spaces: https://en.wikipedia.org/wiki/INI_file

Comment (Uxio0) [reply setup.cfg:4]: `503` was ignored by default in `flake8 < 4`, I don't know why they enabled it as `503` and `504` are incompatible

Comment (Uxio0) [reply setup.cfg:4]: https://docs.python.org/3/library/configparser.html

**Review by moisses89 — APPROVED** [review]

**Pull request merged**